Documentation/DocuBlocks/Rest/Administration/get_admin_metrics_v2.md

```diff
@@ -0,0 +1,46 @@
+
+@startDocuBlock get_admin_metrics_v2
+@brief return the current instance metrics
+
+@RESTHEADER{GET /_admin/metrics/v2, Read the metrics, getMetricsV2}
+
+@RESTQUERYPARAMETERS
+
+@RESTQUERYPARAM{serverId,string,optional}
+Returns metrics of the specified server. If no serverId is given, the asked
+server will reply. This parameter is only meaningful on Coordinators.
+
+@RESTDESCRIPTION
+Returns the instance's current metrics in Prometheus format. The
+returned document collects all instance metrics, which are measured
+at any given time and exposes them for collection by Prometheus.
+
+The document contains different metrics and metrics groups dependent
+on the role of the queried instance. All exported metrics are
+published with the prefix `arangodb_` to distinguish them from other
```

Comment:
This is not 100% correct. We have quite a few which are prefixed with `rocksdb_` and do not have the `arangodb_` prefix. I suggest we add "or `rocksdb_`" to this sentence.

Reply:
Fixed.